_data-prepper/simple-pipelines.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: default
+title: Simple pipelines
+nav_order: 12
+---
+
+# Simple pipelines
+
+This tutorial walks you through creating a simple Data Prepper pipeline and how to configure it.
+
+## Installation
+
+First, install Data Prepper with Docker using the following command:
+
+```
+docker pull opensearchproject/data-prepper:latest
+```
+
+## Simple configuration
+
+To help you get started, we have a simple configuration which you can use to run Data Prepper. It generates random strings and writes them to `stdout`. 
+
+Create a Data Prepper configuration file, `data-prepper-config.yaml` and include the following line:
+
+```
+ssl: true
+```
+
+Create a Data Prepper pipeline file, `pipelines.yaml`, with:
```

Comment:
Rather than "with", we need to provide clear direction, as we did on the preceding line.